cosmos/launchpad/construction_submit.go

```diff
@@ -3,8 +3,9 @@ package launchpad
 import (
 	"context"
 	"encoding/base64"
-	"encoding/hex"
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/simapp"
```

Comment:
Mmm... Do we really need to introduce this dep? Can't we recreate the same coded we have in Launchpad instead?